<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OriginSpacingWidget</class>
 <widget class="QWidget" name="OriginSpacingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Origin</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>x</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QDoubleSpinBox" name="originXSpinbox">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-10000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>y</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QDoubleSpinBox" name="originYSpinbox">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-10000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>z</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QDoubleSpinBox" name="originZSpinbox">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-10000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Spacing</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>x</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QDoubleSpinBox" name="spacingXSpinbox">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-10000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>y</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QDoubleSpinBox" name="spacingYSpinbox">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-10000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>z</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QDoubleSpinBox" name="spacingZSpinbox">
              <property name="decimals">
               <number>6</number>
              </property>
              <property name="minimum">
               <double>-10000000000.000000000000000</double>
              </property>
              <property name="maximum">
               <double>10000000000.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QPushButton" name="performButton">
        <property name="text">
         <string>Perform</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
